### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ Navigation menu _ Item count is not announced 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3148]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Navigation menu
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _ Modify Modules _ Navigation menu _ Item count is not announced 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Now navigate to all the menu items from the left side navigation using arrow keys one at a time.

3. Issue Type: Manual
Observed Behavior:
On accessing all the menu items from the left side navigation using arrow keys one at a time , the respective Value/Item count e.g. ‘1 of 6’  is not announced by the screen reader.
Screenshot:
 

Expected Behavior:
On accessing all the menu items from the left side navigation using arrow keys one at a time , the Value/Item count e.g. ‘1 of 6’   must be announced by the screen reader along with other necessary information as it is a list.

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1 (Level A) _ Modify Modules _ Navigation menu _ Item count is not announced.docx](https://github.com/user-attachments/files/16660103/ACC-264.1.Level.A._.Modify.Modules._.Navigation.menu._.Item.count.is.not.announced.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660105/Issue.processing-09.2021.002.docx)

**Comment by OliwiaGowor** [comment]
Seems like it was fixed with the UI5 migration

<img width="957" alt="Image" src="https://github.com/user-attachments/assets/d580a716-9464-4199-9da8-11d23efeabfa" />
<img width="987" alt="Image" src="https://github.com/user-attachments/assets/6edf663d-cf1e-452a-8638-6d1687c689c9" />
